README.md

```diff
@@ -37,7 +37,14 @@ Read more about jsDeliver versioning on their [website](http://www.jsdelivr.com/
 
 ## Configuration
 
-No adapter configuration is currently available, read the [Chart.js documention](https://www.chartjs.org/docs/latest) for other possible date/time related options. For example, the time scale [`time.*` options](https://www.chartjs.org/docs/latest/axes/cartesian/time.html#configuration-options) can be overridden using the [date-fns tokens](https://date-fns.org/v1.30.1/docs/format).
+Locale support:
+date-fns, contrary to moment.js which uses a global locale configuration, requires the date-fns locale object to be tagged on to each `format()` call, which requires the locale to be explicitly set via the `adapters.date` option: [Chart.js documentation on adapters.date](https://www.chartjs.org/docs/latest/axes/cartesian/time.html?h=adapter)
+eg.
```

Comment:
Need an extra line break before `eg.` (which I would rename for `For example:`)

```
... ian/time.html?h=adapter)

For example:

```javascript
```